projects/plugins/jetpack/extensions/blocks/latex/latex.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Plugin Name:       LaTex
+ * Description:       A WordPress block for live editing and displaying LaTex in posts and pages.
+ * Version:           0.1.0
+ * Author:            Jetpack Team
+ * License:           GPL-2.0-or-later
+ * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain:       jetpack
+ *
+ * @package LatexFormula
+ */
```

Comment:
Let's get rid of all of this, that is no longer necessary in this context.


```suggestion
/**
 * LaTeX Block.
 *
 * @since $$next-version$$
 *
 * @package automattic/jetpack
 */
```